tern/formats/cyclonedx/cyclonedx.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2019-2020 VMware, Inc. All Rights Reserved.
```

Comment:
You can add your own Copyright here `Copyright (c) 2021 Patrick Dwyer` if you want.